packages/flutter_tools/test/integration.shard/test_driver.dart

```diff
@@ -679,8 +679,7 @@ class FlutterTestTestDriver extends FlutterTestDriver {
       'test',
       '--disable-service-auth-codes',
       '--machine',
-      '-d',
-      'flutter-tester',
+      '--start-paused',
```

Comment:
We seem to be adding '--start-paused' argument already at line 89 conditional on withDebugger=true(which is the case for expression evaluation tests).

Reply:
Ahh thanks, I'll remove this then